sfaira/unit_tests/directories.py

```diff
@@ -15,6 +15,8 @@
 DIR_DATA_DATABASES_CACHE = os.path.join(_DIR_DATA_DATABASES, "cache")
 DIR_DATABASE_STORE_DAO = os.path.join(_DIR_DATA_DATABASES, "store_dao")
 
+SFAIRA_REPO_URL = "https://zenodo.org/record/4836517/files/"
```

Comment:
See above, likely needs to explicitly point to the same repo as the UI via `sfaira.consts`

Reply:
done